### DISCUSSION TITLE
Where to correct misspelling of POI?

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#12524] [General]
The Danish translation of the POI 'Restroom' is misspelled. Currently is says 'Toiletetter' but should be 'Toiletter'. I do some work on the OsmAnd localization myself at Weblate but have not been able to find a Restroom string there. So I figure the POI texts are pulled from somewhere else but where?

**Comment by laralem** [comment]
It's in https://hosted.weblate.org/translate/osmand/phrases/da/?checksum=387fae865c35fa
As we can see in https://hosted.weblate.org/languages/da/osmand/  it's in "Phrases" component, related to OpenStreetMap tags translations. The "main" apps interface translations are in "Android" and "iOS" components.
